packages/create-plugin/templates/common/_package.json

```diff
@@ -51,6 +51,7 @@
     "sass-loader": "13.3.1",
     "style-loader": "3.3.3",
     "swc-loader": "^0.2.3",
+    "terser-webpack-plugin": "^5.3.10",
```

Comment:
is this necessary?

Reply:
For yarn pnp it is required for builds to run as without it we access a transitive dependency and yarn errors out with something like:

```
Error: Your application tried to access terser-webpack-plugin, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.
```